repose-aggregator/tests/functional-tests/src/integrationTest/groovy/features/filters/samlpolicy/SamlAttributeMappingTest.groovy

```diff
@@ -462,10 +469,18 @@ class SamlAttributeMappingTest extends ReposeValveTest {
         !json.access.'RAX-AUTH:extendedAttributes'.user."$extAttribPath"
 
         where:
-        [withOut, extAttribPathPolicy, remoteValue] << [
-                ["without", "{0}", [[path: $//saml2p:Response/saml2:Assertion/saml2:Subject/saml2:NameID/@SPProvidedID/$]]],
-                ["with", "{Pt(/saml2p:Response/saml2:Assertion/saml2:Subject/saml2:NameID/@SPProvidedID)}", null]
+        [withOut, extAttribPathPolicy, remoteValue, acceptType] << [
+            ["without", "{0}", [[path: $//saml2p:Response/saml2:Assertion/saml2:Subject/saml2:NameID/@SPProvidedID/$]], APPLICATION_JSON],
+            ["with", "{Pt(/saml2p:Response/saml2:Assertion/saml2:Subject/saml2:NameID/@SPProvidedID)}", null, APPLICATION_JSON],
+            ["without", "{0}", [[path: $//saml2p:Response/saml2:Assertion/saml2:Subject/saml2:NameID/@SPProvidedID/$]], TEXT_YAML],
+            ["with", "{Pt(/saml2p:Response/saml2:Assertion/saml2:Subject/saml2:NameID/@SPProvidedID)}", null, TEXT_YAML]
         ]
+
+//        where:
```

Comment:
Nope. It will be removed here:

* https://github.com/rackerlabs/repose/pull/1781/commits/dd8ed201893992d02e5b8d9bee4a08ab836218d8